evm_loader/tests/contracts/rw_block.sol

```diff
@@ -2,9 +2,9 @@
 pragma solidity >=0.5.12;
```

Comment:
pls change name for file. Because we are using rw_lock but contract file named rw_Block.